go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/tensorleap/leap-cli
 
-go 1.20
+go 1.21
+
```

Comment:
The last time I test got version 1.21 we are not supporting go 1.21 right now - we need to make sure we do not break any functionality, we have some issues on Doker and some other packages, so we need some coverage on the CLI functionality to make sure it works

Reply:
no problem. I'll just revert it now and so we could merge the Makefile changes by themselves.